packages/mrm-task-license/index.js

```diff
@@ -7,18 +7,32 @@ const defaultLicense = 'MIT';
 
 const ANONYMOUS_LICENSES = ['Unlicense'];
 
-const isAnonymouseLicense = name => ANONYMOUS_LICENSES.indexOf(name) > -1;
+const isAnonymousLicense = name => ANONYMOUS_LICENSES.includes(name);
+
+function getAuthorName() {
```

Comment:
I think passing `pkg` or even `pkg.get('author')` instead of reading `package.json` again would make more sense here.

Reply:
Yeah, it makes sense. Thank you for review